packages/next/src/server/lib/start-server.ts

```diff
@@ -264,6 +264,8 @@ export async function startServer(
         maxExperimentalFeatures: 3,
       })
 
+      Log.event(`Starting`)
```

Comment:
The `...` is intended to explain "it's running, please wait"
```suggestion
      Log.event(`Starting...`)
```